src/styles/blocks/card.css

```diff
@@ -0,0 +1,107 @@
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+
+  display: block;
+
+  margin: 0 auto 40px;
+}
+
+.card__product-name {
+  margin-bottom: 4px;
+
+  display: flex;
+  align-items: space-between;
+
+  line-height: 18px;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--color-dark);
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+
+  min-height: 14px;
+
+  margin-bottom: 16px;
+  color: var(--color-gray);
+}
+
+.card__product-rating {
+  margin-bottom: 24px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__product-reviews {
+  min-height: 14px;
+
+  display: flex;
+  align-items: flex-end;
+
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-dark);
+}
+
+.card__price-block {
+  line-height: 18px;
+
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--color-gray);
+}
+
+.card__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--color-dark);
+}
+
+.card__buy-link {
+  width: 166px;
+  min-height: 40px;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
```

Comment:
you need to make transparent border by the task (look closer to №9 in checklist)